source/deployment/on-boarding.rst

```diff
@@ -9,6 +9,11 @@ DO NOT manipulate the Mattermost database
   - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
   - If you need to permanently delete a team or user, use the `Command Line Tool <../administration/command-line-tools.html>`__.
 
+Updating the Support Email & Help Link
+---------------------------------------
+
+Update the Support Email and Help Link in Mattermost to provide your end users a resource for password resets or questions on their Mattermost account.  The Support Email is used on email notifications and during tutorial for end users to ask support questions.  The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system. 
```

Comment:
where are they updated?

Reply:
@jasonblais - thanks for catching, totally forgot to add that!